MantidQt/CustomInterfaces/test/EnggDiffractionPresenterTest.h

```diff
@@ -1128,6 +1134,55 @@ class EnggDiffractionPresenterTest : public CxxTest::TestSuite {
     pres.notify(IEnggDiffractionPresenter::RebinMultiperiod);
   }
 
+  void test_fitting_with_missing_param() {
+    testing::NiceMock<MockEnggDiffractionView> mockView;
+    MantidQt::CustomInterfaces::EnggDiffractionPresenter pres(&mockView);
+
+    pres.notify(IEnggDiffractionPresenter::FitPeaks);
+  }
```

Comment:
Is this test meant to just check that it doesn't crash? I'd always put a few `EXPECT_CALL` to check basic conditions. For example the couple of `EXPECT_CALL` about `userError` and `userWarning`. I imagine in this case it shows a warning/error pop-up?